packages/lib-classifier/src/components/Classifier/components/MetaTools/components/Metadata/components/MetadataModal/MetadataModal.js

```diff
@@ -27,10 +27,13 @@ const DatumWrapper = styled(Text)`
 `
 
 export function formatValue (value) {
+  if (value?.startsWith('http')) {
```

Comment:
🚨 This does not work. Please run [Mark’s test project](https://local.zooniverse.org:3000/projects/markbouslog/test-project-mb/classify/workflow/22300?env=production) locally and double check. I get an error thrown by the MetadataModal component. Suggestion to write a unit test for this change too. 

Here's a link to Mark's project [workflow on staging](https://frontend.preview.zooniverse.org/projects/markbouslog/test-project-mb/classify/workflow/22300) where metadata links open in a new tab.

![Screenshot 2023-10-24 at 1 11 54 PM](https://github.com/zooniverse/front-end-monorepo/assets/23665803/1ec61155-9176-4a63-96f3-266f0a751df6)

Reply:
That's odd. `value` must be a string for the key value pairs in subject metadata, which is serialised as a string by the API, or possibly null if the value is empty. 

This works on every subject I've tested it with. It should work in the storybook too. I thought it already had tests. I'll take a look. 

Either way, metadata values shouldn't be parsed as markdown here. URLs should be rendered as links in JSX.

Reply:
Do you happen to have the subject metadata that's breaking the link here?

Reply:
Did you take a look at the tests? They render the mock subject from the storybook. The metadata includes a URL, and it's rendering without errors. 

I'm really curious to know how a subject has broken this, because we should add the failing case to the storybook. 

https://github.com/zooniverse/front-end-monorepo/blob/dbc9aca77ea8f103d041cff6c3018ab57c2e78be/packages/lib-classifier/src/components/Classifier/components/MetaTools/components/Metadata/components/MetadataModal/MetadataModal.stories.js#L8-L13

Reply:
Bug confirmed on this branch here. 
https://fe-project-branch.preview.zooniverse.org/projects/markbouslog/test-project-mb/classify/workflow/22300

Nothing looks unusual about the subject metadata, though.

Reply:
Could be that the tests here are passing when they should be failing. However, if that were the case I’d expect the storybook to crash too.

Reply:
I've found the bug. I Fancy Cats subjects have numbers in their metadata. The mock subject in the tests only has string metadata.

The metadata changes are big enough that I'm going to move them out into their own PR.
<img width="400" alt="Metadata for an I Fancy Cats subject, which is a mixture of strings and numbers." src="https://github.com/zooniverse/front-end-monorepo/assets/59547/9242a0c1-1752-4293-8d0c-fc315fb24ddc">

Reply:
I've cherry-picked these changes into https://github.com/zooniverse/front-end-monorepo/pull/5525, so that they can be merged while this is still pending review.